src/protect-function-app/README.md

```diff
@@ -0,0 +1,130 @@
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Python 3.8+
+- [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
+- An empty, [Python 3.8 Function App (v4) deployed to Azure](https://docs.microsoft.com/azure/azure-functions/create-first-function-cli-python) and the permissions or role required to modify its settings
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an API application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis#register-the-web-api) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                            | Notes                                                                           |
+|-------------------------------:|:---------------------------------------------------------------------|:--------------------------------------------------------------------------------|
+| **Name**                       | `python-azure-function-api`                                          | Suggested value for this sample. <br/> You can change the app name at any time. |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                              |
+| **Scopes defined by this API** | **Scope name**: `Greeting.Read`<br/>**Who can consent?**: **Admins and users**<br/>**Admin consent display name**: `Read API Greetings`<br/>**Admin consent description**: `Allows the user to see greetings from the API.`<br/>**User consent display name**: `Read API Greetings`<br/>**User consent description**: `Allows you to see greetings from the API.`<br/>**State**: **Enabled** | Required scope for this sample. |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Enable Function App Authentication
+
+1. Open the existing Function App in the Azure portal.
+1. Click on **Authentication** and then **Add identity provider**.
+1. Use these settings in your identity provider configuration.
```

Comment:
```suggestion
### 2. Enable Function app authentication

Next, complete the steps in [Enable Azure Active Directory in your App Service app](https://docs.microsoft.com/azure/app-service/configure-authentication-provider-aad?toc=/azure/azure-functions/toc.json#-enable-azure-active-directory-in-your-app-service-app) to add Azure Active Directory as an identity provider for your API.
```

Let's let the Functions/App Service docs do the heavy lifting on any procedural stuff for those services. Whenever possible, avoid steps in the README sections.

Reply:
Oh, yeah, great idea. I'll tidy this up.